spec/helpers/answers_helper_spec.rb

```diff
@@ -0,0 +1,132 @@
+require "spec_helper"
+
+RSpec.describe AnswersHelper, type: :helper do
+  describe "#answer_items" do
+    it "adds a query string to the link for each item" do
+      helper.answer_items.each do |item|
+        expect(item[:edit][:href]).to include("?change-answer")
+      end
+    end
+
+    context "nhs_number" do
+      it "includes the nhs_number question if it has a value" do
+        session[:nhs_number] = "1234567890"
+        expect(helper.answer_items.pluck(:field)).to include(
+          I18n.t("coronavirus_form.questions.nhs_number.title"),
+        )
+      end
+
+      it "skips the nhs_number question if the value is nil" do
+        expect(helper.answer_items.pluck(:field)).to_not include(
+          I18n.t("coronavirus_form.questions.nhs_number.title"),
+        )
+      end
+    end
+  end
+
+  describe "#concat_answer" do
+    it "returns the answer if the answer is a string" do
+      question = "questions"
+      answer = "answer"
+      expect(helper.concat_answer(answer, question)).to eq(answer)
+    end
+
+    context "contact_details" do
+      let(:question) { "contact_details" }
+
+      it "concatenates contact_details with a line break" do
+        answer = {
+          "phone_number_calls" => "012101234567",
+          "phone_number_texts" => "0777001234567",
+          "email" => "me@me.com",
```

Comment:
Nitpick - can we change this to 'example.com', as the `me.com` address exists and is used.

```suggestion
          "email" => "me@example.com",
```

Reply:
Sure! Added.